modules/mongodb/src/test/java/org/testcontainers/containers/MongoDbContainerTest.java

```diff
@@ -0,0 +1,79 @@
+package org.testcontainers.containers;
+
+import com.mongodb.ReadConcern;
+import com.mongodb.ReadPreference;
+import com.mongodb.TransactionOptions;
+import com.mongodb.WriteConcern;
+import com.mongodb.client.ClientSession;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.TransactionBody;
+import org.bson.Document;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class MongoDbContainerTest {
+    private final MongoDbContainer mongoDbContainer = new MongoDbContainer();
```

Comment:
please use try-with-resources in the test's body

Reply:
done

Reply:
LGTM